vscode-wpilib/resources/gradle/shared/.gitignore

```diff
@@ -169,6 +169,9 @@ out/
 .fleet
 
 # Simulation GUI and other tools window save file
+networktables.json
+simgui.json
+simgui-*.json
```

Comment:
`simgui-ds.json` is usually useful to commit, since it stores button mappings. `simgui-window.json` is covered by `*-window.json`

Reply:
iirc there are more simgui-* files?